pkg/config/config.go

```diff
@@ -756,6 +756,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("runtime_security_config.run_path", defaultRunPath)
 	config.BindEnvAndSetDefault("runtime_security_config.event_server.burst", 40)
 	config.BindEnvAndSetDefault("runtime_security_config.event_server.rate", 10)
+	config.BindEnvAndSetDefault("runtime_security_config.load_controller.events_count_threshold", 20000)
```

Comment:
Could you please open a PR to add these settings to the public helm chart? Thank you!

Reply:
Will do 👍